src/ray/protobuf/experimental/instance_manager.proto

```diff
@@ -17,29 +17,22 @@ option cc_enable_arenas = true;
 
 package ray.rpc.autoscaler;
 
-enum RayNodeKind {
-  UNKNOWN = 0;
-  HEAD = 1;
-  WORKER = 2;
+// A status code to be used for operations in the Instance Manager
+// and other components.
+enum StatusCode {
```

Comment:
Do we need to put it under ray.rpc.autoscaler.im package to avoid conflict? or name it IMStatusCode?